community-membership.md

```diff
@@ -89,8 +89,8 @@ The following apply to the part of the codebase for which one would be an approv
 - Reviewer for, or author of, at least 5 substantial PRs to the codebase,
   with the definition of substantial area to the maintainer's discretion
   (e.g. refactors/adds new functionality rather than one-line pulls)
-- Nominated by a maintainer:
-  - With no objections from other maintainers
+- Nominated by a maintainer from the relevant repository:
+  - With no objections from other repository maintainers
```

Comment:
- How long do you wait for objections? One week?
- What is the resolution process to address objections from maintainers of other repos?
- How many maintainers of the relevant repo must approve? I suggest this to be 2 (1 in case of solo maintainer repos) instead of a majority (there is still a period of objection from all maintainers anyway).
The way this is written now this point isn't clear, and one could even argue this only requires a single maintainer nomination and simply no objections within one week.

Reply:
>  and one could even argue this only requires a single maintainer nomination and simply no objections within one week

This is how it was done until now

Reply:
Ok, that's fine! Then let's add the one week for the objection period in there?

Reply:
> How long do you wait for objections? One week?

Yes, added to text.

> What is the resolution process to address objections from maintainers of other repos?

STC always serves as the escalation point. Added to text.

> How many maintainers of the relevant repo must approve? I suggest this to be 2

So far we've not had approving votes for approvers, but I think we can accommodate that to make the process more consistent with maintainers and approvers. Added to text.